docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-1a.md

```diff
@@ -8,7 +8,12 @@ here.
 ```shell
 kubectl ws root:espw
 mailbox-controller -v=2 &
-sleep 60
+# wait until workspaces for both mailboxes show up
+sleep 10
+kubectl ws root
+while [ $(kubectl ws tree | grep "-mb-" | wc -l) -ne 2 ]; do
```

Comment:
There is a problem in the `grep` command here: the pattern is confused with flags.  See the error message "grep: invalid max count" reported in the failed test of example1.

Reply:
Good catch, adding escape characters to regex.